.github/workflows/ci.yml

```
@@ -107,18 +107,35 @@ jobs:
     strategy:
       matrix:
         go-version: ["~1.22.0", "~1.21.3"]
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        # GitHub Actions does not support arm* architectures on default
-        # runners. It is possible to accomplish this with a self-hosted runner
-        # if we want to add this in the future:
-        # https://docs.github.com/en/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow
-        arch: ["386", amd64]
-        exclude:
-          # Not a supported Go OS/architecture.
+        platform:
+          - os: ubuntu-latest
+            arch: "386"
+          - os: ubuntu-latest
+            arch: amd64
           - os: macos-latest
+            arch: amd64
+          - os: windows-latest
             arch: "386"
-    runs-on: ${{ matrix.os }}
+          - os: windows-latest
+            arch: amd64
+          - os: actuated-arm64-1cpu-1gb
```

Is there any list of what Actuated machines are available? Any docs that would help us in troubleshooting?

E.g. why not

```suggestion
          - os: actuated-arm64
```

---

It's not yet clear to me what the default CPU/memory would be if we don't specify any. I'll ask the actuated folks.

---

They told me we shouldn't use the default value, and always specify explicit ones.

---

Any reasons why? Are there any docs?

Their docs suggest something else: https://docs.actuated.dev/test-build/

---

Note that this discussion happened in the actuated slack.
I don't think they have an auto-invite system. But if you ping me on slack, I can invite you.